docs/pages/docs/handbook/package-installation.mdx

```diff
@@ -0,0 +1,180 @@
+import Callout from "../../../components/Callout";
+import HeartIcon from "@heroicons/react/solid/HeartIcon";
+import { Tabs, Tab } from "../../../components/Tabs";
+
+# Package Installation
+
+A package manager (like `npm`) handles two things for you: installing packages and [managing workspaces](/docs/handbook/workspaces).
+
+Turborepo is compatible with three package managers:
+
+- [npm](https://docs.npmjs.com/cli/v8/using-npm/workspaces#description)
+- [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
+- [pnpm](https://pnpm.io/workspaces)
+
+## Installing packages
+
+When you first clone or create your monorepo, you'll need to:
+
+1. Make sure you're in the root directory of your monorepo
+2. Run the install command:
+
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
```

Comment:
A canonical reference could also let us turn this into something like this, so we don't need to specify `items` everywhere and the component can also require examples for all supported clients

```jsx
<NpmClientUsageTabs>
</NpmClientUsageTabs>
```